modules/charon-core/src/test/java/org/wso2/charon3/core/protocol/endpoints/UserResourceManagerTest.java

```diff
@@ -162,42 +504,797 @@ public void testListWithGetInt(String filter, int startIndexInt, int countInt,
 
     @Test(dataProvider = "dataForListWithGetInteger")
     public void testListWithGetInteger(String filter, Integer startIndexInt,
-                        Integer countInt, String sortBy, String sortOrder, String domainName,
-                        String attributes, String excludeAttributes, int expectedScimResponseStatus) {
-
+                                       Integer countInt, String sortBy, String sortOrder, String domainName,
+                                       String attributes, String excludeAttributes, int expectedScimResponseStatus) {
 
         SCIMResponse outputScimResponse = userResourceManager.listWithGET(userManager, filter, startIndexInt,
                 countInt, sortBy, sortOrder, domainName, attributes, excludeAttributes);
         Assert.assertEquals(outputScimResponse.getResponseStatus(), expectedScimResponseStatus);
     }
 
-    private User getNewUser() throws BadRequestException, CharonException, InternalErrorException {
+    @DataProvider(name = "dataForTestCreateUserSuccess")
+    public Object[][] dataToTestCreateUserSuccess() {
+
+        return new Object[][]{
+                {NEW_USER_SCIM_OBJECT_STRING, "userName", null},
+                {NEW_USER_SCIM_OBJECT_STRING, "userName", "emails"}
```

Comment:
add dataset with both `attributes` and `excludeAttributes` having null values

Reply:
Added in commit 6f5ca461fa07ac4b49182320325156a58b80b910